cross-chain/sei/deploy/05_deploy_l1btc_depositor_ntt_executor_sepolia.ts

```diff
@@ -0,0 +1,44 @@
+import type { HardhatRuntimeEnvironment } from "hardhat/types"
+import type { DeployFunction } from "hardhat-deploy/types"
+import {
+  deployL1BTCDepositorNttWithExecutor,
+  NETWORK_CONFIGS,
+} from "../scripts/deploy-l1btc-depositor-ntt-executor"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
```

Comment:
Does it make sense to verify any prerequisites before we deploy our contracts? Given that we use external dependencies